src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -46,5 +46,9 @@
             <argument type="service" id="request_stack" />
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="validate_request_listener" class="Symfony\Component\HttpKernel\EventListener\RequestListener">
```

Comment:
Not really, as `request_listener` can easily conflict with an existing service. But having a disconnect between the class name and the id is strange.